src/components/CommandUtils.ts

```diff
@@ -50,13 +52,13 @@ export const defaultCommandPatterns: CommandDefaultArgSpec = {
     ]
 };
 
-export const commandDefaults: CommandDefaultArgs = {
+export const commandDefaults: CommandDefaultArgs = {//
```

Comment:
lots of duplication here between Type name and variable name, along with concept, feels messy